papyrus-input.html

```diff
@@ -41,8 +41,7 @@
       }
 
       label {
-        display: block;
-        margin-bottom:5px;
+        pointer-events: none;
```

Comment:
As per conversation - this is an extra fix for non-shadow dom browsers that don't encapsulate id attributes, so that clicking on label doesn't cause user to jump to fist element on page.

Restoring the label click behavior fully would be more effort.